client/js/lounge.js

```diff
@@ -465,6 +468,22 @@ $(function() {
 		}
 	}
 
+	function updateDisconnectedUsers(channel, message) {
+		switch (message.type) {
+		case "nick":
+			channel.find(".msg .user[data-name='" + message.from + "']").addClass("disconnected");
+			channel.find(".msg .user[data-name='" + message.new_nick + "']").removeClass("disconnected");
+			break;
+		case "join":
+			channel.find(".msg .user[data-name='" + message.from + "']").removeClass("disconnected");
+			break;
+		case "part":
```

Comment:
If you could just add a comment `// Deliberate fall through` just so it's clear to people in the future.